fstab-check.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+mapfile -t mountpoints < <(awk '$1 !~ /^#/ && $2 ~ /^[/]/ {print $2}' /etc/fstab)
+for mount in "${mountpoints[@]}"
+do
+   if ! findmnt "$mount" &> /dev/null
+   then
+         echo "node_fstab_mount_status{filesystem=\"$mount\"} 0" >&1
```

Comment:
Redirecting stdout to file descriptor 1 is redundant, since this script does not change the default stdin / stdout / stderr FDs.
```suggestion
         echo "node_fstab_mount_status{filesystem=\"$mount\"} 0"
```